spec/section/joins.md

```diff
@@ -1,4 +1,4 @@
-## RML Joins {#joins}
+## Joins {#joins}
 
 A <dfn>join</dfn> (`rml:Join`) is an operation that extends the logical iteration of one logical view (the child logical view) with fields from another logical view (the parent logical view).
```

Comment:
We should probably define a new class for these joins, since `rml:Join` already exists for referencing object maps. Proposal `rml:ViewJoin`

Reply:
added to agenda TF meeting